core/internal/testutils/configtest/general_config.go

```diff
@@ -579,6 +589,20 @@ func (c *TestGeneralConfig) GlobalEthTxReaperThreshold() (time.Duration, bool) {
 	return c.GeneralConfig.GlobalEthTxReaperThreshold()
 }
 
+func (c *TestGeneralConfig) GlobalEvmEIP1559DynamicFees() (bool, bool) {
```

Comment:
I think we could get some great coverage by enabling this in a bunch of integration tests which use the geth sim, then you'd basically be testing all our features against a real geth node. Wdyt?

Reply:
Yup I agree. Which integration tests do you have in mind?

Reply:
I'd say TestIntegration_OCR, TestIntegration_DirectRequest,  TestFluxMonitor_Deviation, TestIntegration_VRF_JPV2 and TestKeeperEthIntegration

Reply:
Good call, I realised this isn't compatible with Keeper yet.